src/c_api.rs

```diff
@@ -465,6 +465,33 @@ pub unsafe extern "C" fn c2pa_reader_resource_to_stream(
     }
 }
 
+/// Writes all the resources in a Reader to a folder.
+///
+/// # Parameters
+/// * reader_ptr: pointer to a Reader.
```

Comment:
I like to add if the memory pointed to is actually modified or used read-only.
Here it's read-only, so not sure if it's worth adding?